sormas-ui/src/test/java/de/symeda/sormas/ui/externalmessage/RelatedLabMessageHandlerTest.java

```diff
@@ -13,7 +13,7 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
-package de.symeda.sormas.ui.labmessage;
+package de.symeda.sormas.ui.externalmessage;
```

Comment:
If the tested class gets moved (based on a review comment, this test class also needs to be moved.